src/Widget/Field.php

```diff
@@ -741,17 +747,19 @@ private function addInputId(): string
      *
      * @param FormModelInterface $data Form model.
      * @param string $attribute Form model property this widget is rendered for.
-     * @param array $options The HTML attributes for the widget container tag.
+     * @param array $options The HTML attributes for the widget container tag if it is enabled.
      * See {@see \Yiisoft\Html\Html::renderTagAttributes()} for details on how attributes are being rendered.
+     * @param bool $containerEnabled whether the widget container is enabled. Default: true
      *
      * @return self
      */
-    public function config(FormModelInterface $data, string $attribute, array $options = []): self
+    public function config(FormModelInterface $data, string $attribute, array $options = [], $containerEnabled = true): self
```

Comment:
I think a containerEnabled() method would be better.

Reply:
@terabytesoftw I first though also about adding a method, but it would be weird to have an ability to do `containerEnabled(true)` if it is already enabled by default. It would make sense to have a separate method if we also `$options` there too. In that case we will remove `$options` from `config` method and add `container` method with 2 arguments `function container($enabled = true, array $options = [])`. What do you think?

Reply:
If `$options` for the container are configured in `config()` method and enabling/disabling it is a separate method, it becomes weird to have an ability to set the options in config and then disable the container. So I think it is better to keep these 2 things together either in `config` method as it is now or a separate method with both of them.

Reply:
We can change the name to `enclosedByContainer()` and in the same way pass the options for the container.